tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/ArchetypesTest.java

```diff
@@ -72,6 +104,17 @@ public static void setup() {
 		String mvnConfigFileName = new File(MAVEN_SETTINGS_PATH).getAbsolutePath();
 		WorkbenchPreferenceDialog preferenceDialog = new WorkbenchPreferenceDialog();
 		preferenceDialog.open();
+
+		TreeItem t = new DefaultTreeItem(new DefaultTree(preferenceDialog), "Java", "Installed JREs");
```

Comment:
You should use RedDeer API: https://github.com/eclipse/reddeer/blob/master/plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/jdt/debug/ui/jres/JREsPreferencePage.java#L35

Reply:
Fixed.